app/containers/message/Urls.tsx

```diff
@@ -108,8 +108,16 @@ const Url = React.memo(
 		useEffect(() => {
 			if (!url.image && url.url) {
 				const testImageUrl = async () => {
-					const result = await isImageURL(url.url);
-					setIsImageUrlFromParamUrl(result);
+					if (isIOS) {
+						const result = await isImageURL(url.url);
+						setIsImageUrlFromParamUrl(result);
+					} else {
+						Image.getSize(
+							url.url,
+							() => setIsImageUrlFromParamUrl(true),
+							() => setIsImageUrlFromParamUrl(false)
+						);
+					}
```

Comment:
The function isImageURL is returning some difference between platforms, as example the URL https://play.google.com/apps/test/RQVpXLytHNc/ahAETVdb4gnoB4cOV8_fm8D0Vkswqyp2fluIarrP5O3OpKyh-oAUEiuNV6zfOvGHuI3crF0LIvu5Bh26-Boz0eF0KP :

Android <image src="https://user-images.githubusercontent.com/47038980/214647799-e258bede-2c06-4a0c-8f2f-6e54f0d87824.png" width="200" /> iOS <image src="https://user-images.githubusercontent.com/47038980/214647823-7164b144-c1ea-4764-a498-9d77835ea0f9.png" width="200" />

It should not show any line or preview

Using the Image.getSize for Android, I was able to reproduce the same behavior for Android and iOS.

Also, both platforms aren't able to render any image with .svg format
example: https://global-uploads.webflow.com/611a19b9853b7414a0f6b3f6/6141e37299de7d330c3699c2_graduation-cap%201.svg

Reply:
SVG's can be rendered with expo-image apparently https://docs.expo.dev/versions/unversioned/sdk/image
Different scope and we should give it a try on another task